testing/load-aware/config.yaml

```diff
@@ -96,3 +102,11 @@ matbench:
   ignore_exit_code: true
   # directory to plot. Set by testing/common/visualize.py before launching the visualization
   test_directory: null
+load_aware:
```

Comment:
can you move this block before `matbench`? it's just out of consistency with the other config files. The order is: cluster settings, RHODS settings if applicable, test settings, visualization settings, you get the idea